src/types/Error.ts

```diff
@@ -0,0 +1,7 @@
+export enum Error {
```

Comment:
Good idea, maybe i litle-bit steal that 👀

Reply:
Your welcome and thanks for review=)